__tests__/app.test.js

```diff
@@ -317,8 +311,46 @@ describe("POST /api/articles/:article_id/comments", () => {
       .post("/api/articles/not-an-article/comments")
       .send(commentOnNonExistentArticle)
       .expect(400)
-      .then((response) => {
-        expect(response.body.msg).toBe("bad request");
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("bad request");
+      });
+  });
+});
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("DELETE:204 deletes the specified comment and sends no body back", () => {
+    return request(app)
+      .delete("/api/comments/7")
+      .expect(204)
+      .then(() => {
+        return request(app)
+          .get("/api/articles/1/comments")
+          .then(({ body: { comments } }) => {
+            expect(comments).not.toContain({
```

Comment:
Will always pass as 204 response will never contain anything

Reply:
I think I've fixed it!